auth/build.gradle

```diff
@@ -23,7 +23,10 @@ android {
     }
 
     testOptions {
-        unitTests.includeAndroidResources = true
+        unitTests {
+            includeAndroidResources = true
+            returnDefaultValues = true
```

Comment:
I did this because of http://tools.android.com/tech-docs/unit-testing-support#TOC-Method-...-not-mocked.- errors.

Not sure if it's okay but seemed easier than mocking and DI-ing JSONObject and Looper.